notebooks/add_new_data.ipynb

```diff
@@ -0,0 +1,172 @@
+{
```

Comment:
<p>What bucket are you using for this data? I can't find a <code>cdolfi</code> subdir on the DH-PLAYPEN instance.</p><p> </p><p>Also the naming convention you are using is a bit confusing. I think <code>s3_bucket_name</code>   should be something like <code>s3_prefix</code>  since its the directory name you want to use inside of the bucket, right? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/fedora-mailing-list-analysis/pull/11/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>